shallow_backup/utils.py

```diff
@@ -12,24 +12,30 @@ def run_cmd(command):
 	"""
 	try:
 		if not isinstance(command, list):
-			process = sp.run(command.split(), stdout=sp.PIPE)
+			process = sp.run(command.split(), stdout=sp.PIPE, stderr=sp.DEVNULL)
 			return process
 		else:
-			process = sp.run(command, stdout=sp.PIPE)
+			process = sp.run(command, stdout=sp.PIPE, stderr=sp.DEVNULL)
 			return process
 	except FileNotFoundError:  # If package manager is missing
 		return None
 
 
-def run_cmd_write_stdout(command, filepath):
+def run_cmd_write_stdout(command, filepath, package):
```

Comment:
Since this function is only used for package manager list/install commands, we should:

1. Maybe rename the method to something more descriptive?
2. Check if the package manager is installed (before doing anything?) and print an error if it's not.

Any other improvements you see that we could make?

Reply:
For:
1. Maybe `process_package_cmd()`?
2. I mean I think the current way the function's set up is fine for checking if package managers are installed. If we were to add in a separate existence check, that would be 2 subprocess calls for every installed package manager when we're already doing that and executing the appropriate list commands in the same step.

For other improvements can we please change the `Would you like to update shallow-backup path` prompt to have `No` before `Yes`? The path doesn't change often so it's a bit annoying to press down all the time.

Reply:
> For other improvements can we please change the Would you like to update shallow-backup path prompt to have No before Yes? The path doesn't change often so it's a bit annoying to press down all the time.

Lmao yes, please do this.

Reply:
> I mean I think the current way the function's set up is fine for checking if package managers are installed. If we were to add in a separate existence check, that would be 2 subprocess calls for every installed package manager when we're already doing that and executing the appropriate list commands in the same step.

This is a fair point. Let me think about how this should work a bit more...